content/platform/hpe-simplivity/aside.md

```diff
@@ -2,72 +2,72 @@
 title: Simplivity
 version: v 6.01.8964
 description: Simplivity aside links
-tags: ["Simplivity"]
+tags: ["hpe-simplivity"]
 frontpage: false
 isAside: true
 ---
 ### About the API
 
-#### [**What's New**](/platform/hpe-simplivity/whats-new/)
+[**What's New**](/platform/hpe-simplivity/whats-new/)  
 
-#### [**Feature and function support by REST API version**](/platform/hpe-simplivity/feature-and-function-support-by-rest-api-version)
+[**Feature and function support by REST API version**](/platform/hpe-simplivity/feature-and-function-support-by-rest-api-version)  
 
-#### [**Architecture and object model**](/platform/hpe-simplivity/architecture-and-object-model)
+[**Architecture and object model**](/platform/hpe-simplivity/architecture-and-object-model)  
 
-#### [**Enumerations**](/platform/hpe-simplivity/enumerations)
+[**Enumerations**](/platform/hpe-simplivity/enumerations)  
 
-#### [**Interactive REST API reference**](/platform/hpe-simplivity/interactive-rest-api-reference)
+[**Interactive REST API reference**](/platform/hpe-simplivity/interactive-rest-api-reference)  
 
-#### [**About REST API errors and exceptions**](/platform/hpe-simplivity/errors-and-exceptions)
+[**About REST API errors and exceptions**](/platform/hpe-simplivity/errors-and-exceptions)  
 
-#### [**HTTP error code reference**](/platform/hpe-simplivity/http-error-code-reference)
+[**HTTP error code reference**](/platform/hpe-simplivity/http-error-code-reference)  
 
-#### [**Authorization-related REST API log messages and responses**](/platform/hpe-simplivity/authorization-related-rest-api-log-messages-and-responses)
+[**Authorization-related REST API log messages and responses**](/platform/hpe-simplivity/authorization-related-rest-api-log-messages-and-responses)  
 
 ### Explore the API
 
-#### [**HPE OmniStack REST API Reference**](https://developer.hpe.com/api/simplivity/)
+[**HPEOmniStack REST API Reference**](https://developer.hpe.com/api/simplivity/)  
```

Comment:
This is not rendering properly in the aside. 

![Screen Shot 2019-11-14 at 3 42 56 PM](https://user-images.githubusercontent.com/5983843/68894585-91238600-06f5-11ea-9852-32e687a0a0f1.png)

Reply:
gotcha, should be fixed now.